contrib/depends/packages/readline.mk

```diff
@@ -9,6 +9,7 @@ define $(package)_set_vars
   $(package)_build_opts=CC="$($(package)_cc)"
   $(package)_config_env=AR="$($(package)_ar)" RANLIB="$($(package)_ranlib)" CC="$($(package)_cc)"
   $(package)_config_env_darwin=RANLIB="$(host_prefix)/native/bin/x86_64-apple-darwin11-ranlib" AR="$(host_prefix)/native/bin/x86_64-apple-darwin11-ar" CC="$(host_prefix)/native/bin/$($(package)_cc)"
+  $(package)_config_env_android=LDFLAGS="-L$(host_prefix)/lib"
```

Comment:
The built readline needs to find and link against the previously built ncurses. Not sure why this ever worked on other platforms, before.

Reply:
This highlights a bug, this LDFLAGS should be set on all platforms. The current builds aren't using the ncurses we built, on Linux or other OSs. This has been broken since 496cd46d8097d8363428e2d75e16c889299b02c7

Reply:
Rather than fix this here, I moved the fix to #6081